superset-frontend/src/views/CRUD/data/dataset/AddDataset/DatasetPanel/DatasetPanel.tsx

```diff
@@ -19,10 +19,12 @@
 import React from 'react';
 import { supersetTheme, t, styled } from '@superset-ui/core';
```

Comment:
@lyndsiWilliams I missed this when we originally worked on DatasetPanel but using supersetTheme directly is an anti-pattern.  We should remove this import, and instead bring in useTheme
`import { useTheme, t, styled } from '@superset-ui/core';`
At line 131-148, and line 288, anywhere else we directly use supersetTheme should be changed.

Reply:
FIxed in [`this commit`](https://github.com/apache/superset/pull/22136/commits/86549660bd4d6ac53e0b7723ab3e9370e37d2a28).